doc/faq.rst

```diff
@@ -50,7 +50,7 @@ it is also working on PyPy.
 2.4 What versions of Python is Pylint supporting?
 --------------------------------------------------
 
-Since Pylint 1.4, we support only Python 2.7+ and Python 3.3+. If code
+Since Pylint 1.8, we support only Python 2.7 and Python 3.4+. If code
```

Comment:
Or should this be 2.0 if following semver?

Reply:
Technically, yes, but `python_requires` kind of mitigates.